BITSWAP.md

```diff
@@ -196,6 +197,94 @@ message Message {
 }
 ```
 
+## Bitswap 1.3.0
+
+Bitswap 1.3.0 extends the Bitswap 1.2.0 protocol with the following changes:
+1. Having a list of tokens that may be sent with the message and referenced within the message
+2. Allowing each entry in the wantlist to contain a set of tokens
+3. Allowing responses (both BlockPresences and Blocks) to contain a set of tokens
+4. Adding an `AuthRequired` BlockPresence
```

Comment:
given that tokens are pretty generic, why not make this error more generic?

Reply:
Maybe just `TokenError`?

Reply:
Done in https://github.com/ipfs/specs/pull/270/commits/661ab5bf195a6d7094851dbec85545cc8052e9bb